Makefile.am

```diff
@@ -10,3 +13,39 @@ AM_DISTCHECK_CONFIGURE_FLAGS = --with-systemdsystemunitdir='$${libdir}/systemd/s
 pkgconfigdir = $(libdir)/pkgconfig
 
 pkgconfig_DATA = eos-updater-0.pc
+
+# Utility library
+noinst_LTLIBRARIES += libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la
```

Comment:
Strictly, no. However, it doesn’t hurt, and I think it’s better to add versioning (and all the other library trimmings) in from the beginning rather than adding them in half-way through the life of a library and omitting something. I plan to slowly add versioning to the other libraries in the project.